tests/test_geometry.scad

```diff
@@ -29,42 +167,28 @@ module test_point_on_segment2d() {
     assert(point_on_segment2d([ 10, 10], [[-10,-10], [10,10]]) == true);
     assert(point_on_segment2d([ 15, 15], [[-10,-10], [10,10]]) == false);
 }
-test_point_on_segment2d();
+*test_point_on_segment2d();
```

Comment:
Does the module actually execute, but have the geometry nullified by `*`, or does it not run at all?

Reply:
Yes. The * prevents the module run at all. Instead of reordering the modules, I have preferred to do the ordered calls at the beginning. The * are there for the case someone prefers to change it to the previous schema.